mozilla/fx-private-relay
# MPP-1898 - Relay number confirmation modal

This PR fixes MPP-1898 

# New feature description

A user must confirm they want this phone number with a confirmation modal.

# Screenshot (if applicable)

<img width="633" alt="image" src="https://user-images.githubusercontent.com/3924990/188227632-e12641f3-7d21-4789-bf5c-db8119c26612.png">

# How to test
1. Navigate to the phones page
2. Make sure you have a real phone verified
3. Make it to the step where you have to choose a relay number
4. Choose a relay number and confirm.
5. You should get a confirmation modal. 

# Checklist
